src/utils/searchUtils.ts

```diff
@@ -259,6 +260,29 @@ export const sortOptions: Record<string, string> = {
   date_desc: "Date (New to Old)",
 }
 
+/**
+ * mapQueryFiltersToSearchFilters
+ * Maps the filter query params to a SearchFilters object
+ */
+export function mapQueryFiltersToSearchFilters(
```

Comment:
This method is fine, but there [already is a method that does exactly this](https://github.com/NYPL/research-catalog/blob/15b7fadeda8c299d898a3348f54663975e115997/src/utils/refineSearchUtils.ts#L11). I don't care which one ends up sticking around, but we should only have one method that converts filter params to an object. Can you pick one and replace the instances of the other with it?

Reply:
I am leaning towards using the existing function since it's already used elsewhere.

Because we want to try to get this out to QA by next week when Diego and Chris are back, I can update this PR to use the existing function. Sounds good, @charmingduchess ?

Reply:
oh dang i totally missed this comment. my bad!